libraries/GCS_MAVLink/MAVLink_routing.cpp

```diff
@@ -122,6 +123,105 @@ bool MAVLink_routing::check_and_forward(mavlink_channel_t in_channel, const mavl
         return true;
     }
 
+#if HAL_WITH_ESC_TELEM
```

Comment:
this is really in the wrong place, should be in AP_ESC_Telem_Backend, and only when enabled

Reply:
Hi @tridge, Could you explain to me why these lines are in the wrong place?  I have built this code and It works fine. I'm sorry. I'm a newbie for this.

Reply:
these messages are not targeted to a specific sysid, so I don't think they are safe to use in this way.
imagine 2 vehicles on the radio link, both generating these msgs, they would consume each others esc telem msgs and get the wrong RPMs to notch filter